backend/src/main/java/de/roman/toolio/controller/InquiryPartController.java

```diff
@@ -23,9 +23,15 @@ public InquiryPartController(InquiryPartService inquiryPartService) {
         return inquiryPartService.listInquiryParts();
     }
 
+    @GetMapping("{id}")
+    public InquiryPart getInquiryById(@PathVariable String id){
+        return inquiryPartService.getInquiryById(id);
+    }
+
     @PostMapping
     public InquiryPart addInquiry(@RequestBody InquiryPart inquiryPartToBeAdded){
-        return this.inquiryPartService.addInquiry(inquiryPartToBeAdded);
+        // TODO Replace hard coded "5" with functionality
+        return this.inquiryPartService.addInquiry(inquiryPartToBeAdded, "5");
```

Comment:
This is the user ID which I have hard coded at the moment. I hope to change the "5" to be the current appuserID. This will be implemented once the entire login logic is created.